app/schema/portal/edit/change-status.js

```diff
@@ -7,6 +7,13 @@ const changeStatusSchema = Joi.object({
   })
 }).required()
 
+const duplicateMicrochipSchema = Joi.object({
+  indexNumber: Joi.string().forbidden().messages({
+    '*': 'The microchip number is in use on another record.'
```

Comment:
I think the wording is due to change here

Reply:
this is what was communicated in the meeting.